compiler/rustc_parse/src/parser/item.rs

```diff
@@ -841,7 +841,17 @@ impl<'a> Parser<'a> {
                 self.bump();
                 Ok(ident)
             }
-            _ => self.parse_ident(),
+            _ => self.parse_ident_common(true),
```

Comment:
Unnecessary change.

Reply:
Fixed on newer commits

Reply:
Perhaps you forgot to push those newer commits?